### PR TITLE
Redirection for option page

### PR DESCRIPTION
**Pull request by mitra-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by habib-deriv — APPROVED** [review]

**Review by mayuran-deriv — APPROVED** [review]

File: src/pages/trade-types/options/digital-options/up-and-down/index.tsx [hunk 1]
```diff
@@ -1,9 +1,19 @@
 import React from 'react'
+import { Helmet } from 'react-helmet'
 import { WithIntl } from 'components/localization'
 import UpAndDownDigitalOptionsPage from 'features/pages/options/digital/up-down'
 
 const DigitalOptions = () => {
-    return <UpAndDownDigitalOptionsPage />
+    const canonicalUrl = '/trade-types/options/digital-options/up-and-down'
```
Comment (ali-hosseini-deriv) [inline src/pages/trade-types/options/digital-options/up-and-down/index.tsx:7]: i recommend to use `window.location.origin` before this, as there's no certainty of how search engines crawlers handle relative path.

**Review by hasan-deriv — APPROVED** [review]

File: gatsby-node.js [hunk 2]
```diff
@@ -155,6 +156,20 @@ exports.onCreatePage = ({ page, actions }) => {
...
+            redirectInBrowser: true,
+            isPermanent: true,
+        })
+        createRedirect({
+            fromPath: `/trade-types/option`,
+            toPath: `/trade-types/options/digital-options/up-and-down/`,
+            redirectInBrowser: true,
+            isPermanent: true,
+        })
+    }
```
Comment (prince-deriv) [inline gatsby-node.js:172]: @mitra-deriv  did we accidentally duplicate this?

Comment (fasihali-deriv) [reply gatsby-node.js:172]: @prince-deriv `fromPath` is different but not sure whether its correct or not. 👀

File: src/pages/trade-types/options/digital-options/up-and-down/index.tsx [hunk 3]
```diff
@@ -1,9 +1,22 @@
 import React from 'react'
+import { Helmet } from 'react-helmet'
 import { WithIntl } from 'components/localization'
 import UpAndDownDigitalOptionsPage from 'features/pages/options/digital/up-down'
+import { isBrowser } from 'common/utility'
 
 const DigitalOptions = () => {
-    return <UpAndDownDigitalOptionsPage />
+    const canonicalUrl = isBrowser()
+        ? `${window.location.origin}/trade-types/options/digital-options/up-and-down`
```
Comment (fasihali-deriv) [inline src/pages/trade-types/options/digital-options/up-and-down/index.tsx:9]: Hi @mitra-deriv are we doing this just UP and Down?

File: gatsby-node.js (hunk 2 above)
Comment (mitra-deriv) [reply gatsby-node.js:172]: yes as @fasihali-deriv its different ,  one is with trailing slash and one is without , the same conventions going for other pages we redirecting

File: src/pages/trade-types/options/digital-options/up-and-down/index.tsx (hunk 3 above)
Comment (mitra-deriv) [reply src/pages/trade-types/options/digital-options/up-and-down/index.tsx:9]: Hi yes @fasihali-deriv cause the old option page gonna redirect to this path only and this is the default tab

File: gatsby-node.js (hunk 2 above)
Comment (fasihali-deriv) [reply gatsby-node.js:172]: @mitra-deriv first one has `s` at the end and second one doesn’t contain `s`. 
please check this 😬

File: src/pages/trade-types/options/digital-options/up-and-down/index.tsx (hunk 3 above)
Comment (fasihali-deriv) [reply src/pages/trade-types/options/digital-options/up-and-down/index.tsx:9]: Also one thing can we use `getDomainUrl()`. 
I am not sure just asking 😬  🙏

File: gatsby-node.js (hunk 2 above)
Comment (mitra-deriv) [reply gatsby-node.js]: @fasihali-deriv thanks for noticing bro , fixed 🙏

**Review by eduard-deriv — APPROVED** [review]

**Comment by eduard-deriv** [comment]
LGTM!

File: src/pages/trade-types/options/digital-options/up-and-down/index.tsx (hunk 3 above)
Comment (mitra-deriv) [reply src/pages/trade-types/options/digital-options/up-and-down/index.tsx:9]: @fasihali-deriv The `getDomainUrl()` only get the hostname , like deriv.com with out https:// , I think we can go ahead with current implementation , thanks  🙏

**Review by fasihali-deriv — APPROVED** [review]

**Review by ali-hosseini-deriv — APPROVED** [review]

**Pull request merged**